index.html

```diff
@@ -1,67 +1,271 @@
+        </li>
+        <li class="mozilla margin-left-5 margin-top-lg">
+          <img src="assets/images/Vector.png" alt="" />
+        </li>
+      </ul>
+    </div>
+    <div class="group1 pos-absolute d-none-lg"></div>
+    <!-- <div class="group2 pos-absolute d-none d-block-lg"></div> -->
+  </section>
+
+  <!-- work section -->
+  <section id="work-section" class="work-section">
+    <article class="card-work">
+      <div class="card-header">
+        <h1 class="card-header-name">My Recent Works</h1>
+        <div class="card-header-border"></div>
+      </div>
+      <div class="cards">
+        <div class="card-img"><img src="assets/images/Img_Placeholder1.png" alt="fantastic-card-image" width="290">
+        </div>
+        <div class="card-name">
+          <h2>Multi-Post Stories</h2>
+        </div>
+        <div class="card-text">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+            industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
       </div>
-      <div class="media-icons">
-        <ul class="list-style-none d-flex d-block-lg">
-          <li class="github"><img src="assets/images/github.png" alt="" /></li>
-          <li class="linkedin margin-left-5 margin-top-lg">
-            <img src="assets/images/linked_in.png" alt="" />
-          </li>
-          <li class="sure margin-left-5 margin-top-lg">
-            <img src="assets/images/sure.png" alt="" />
-          </li>
-          <li class="twitter margin-left-5 margin-top-lg">
-            <img src="assets/images/twitter.png" alt="" />
-          </li>
-          <li class="mozilla margin-left-5 margin-top-lg">
-            <img src="assets/images/Vector.png" alt="" />
-          </li>
+      <div class="card-skills">
+        <ul class="skills d-flex list-style-none">
+          <li class="skill1">css</li>
+          <li class="skill2">html</li>
+          <li class="skill3">bootstrap</li>
+          <li class="skill4">Ruby</li>
         </ul>
+        <button class="card-button">See Project</button>
+      </div>
+    </article>
+    <article class="card-work">
+      <div class="cards card-background-image">
+        <div class="card-name">
+          <h2 class="card-headline">Profesional Art Printing Data</h2>
+        </div>
+        <div class="card-text">
+          <p class="card-headline-text">A daily selection of privately personalized reads; no accounts or sign-ups
+            required. has been the industry's standard</p>
+        </div>
+        <div class="card-skills">
+          <ul class="skills d-flex list-style-none">
+            <li class="skill2 skillcard">html</li>
+            <li class="skill3 skillcard">bootstrap</li>
+            <li class="skill4 skillcard">Ruby</li>
+          </ul>
+          <button class="card-button btn-cards">See Project</button>
```

Comment:
- [ ] Kindly review the Hover of the `See project` button to make sure it is working as per the screenshot below. When I Hover over the button, I don't have the same experience as per Figma design. In the Hover of this button, apply a `background: #FFF5E1;`

![image](https://user-images.githubusercontent.com/62849872/143582748-16cf5a8a-056d-4934-b359-c0075ef46aca.png)